content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -129,6 +129,24 @@ cloudprovider_gce_api_request_duration_seconds { request = "detach_disk"}
 cloudprovider_gce_api_request_duration_seconds { request = "list_disk"}
 ```
 
+
+### kube-scheduler metrics
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+The scheduler exposes optional metrics that reports the requested resources and the desired limits of all running
+pods. These metrics can be used to build capacity planning dashboards, assess current or historical scheduling limits,
+quickly identify workloads that cannot schedule due to lack of resources, and compare actual usage to the pod's request.
+Every pod reports one metrics series for every non-zero resource request or limit. The pod namespace, name, scheduled
+node, priority, and assigned scheduler are reported as labels. Once a pod reaches completion (has a `restartPolicy` of
+`Never` or `OnFailure` and is in the `Succeeded` or `Failed` pod phase, or has been deleted and all containers have a
+terminated state) the series is no longer reported since the scheduler is now free to schedule other pods to run. The
+two metrics are called `kube_pod_resource_request` and `kube_pod_resource_limit`.
+
+The metrics are exposed at the HTTP endpoint `/metrics/resources` and require the same authorization as the `/metrics`
+endpoint on the scheduler. You must use the `--show-hidden-metrics-for-version` flag to expose these alpha stability metrics.
```

Comment:
I would move
>  You must use the `--show-hidden-metrics-for-version` flag to expose these alpha stability metrics.

to just below the heading “kube-scheduler metrics”.

Reply:
What value should the cluster admin set to enable these metrics?

Reply:
Sorry, `1.20` so `--show-hidden-metrics-for-version=1.20` (unless I'm misreading the flag)